examples/data-objects/vltava/package.json

```diff
@@ -51,6 +51,7 @@
     "@fluidframework/request-handler": "^0.29.0",
     "@fluidframework/runtime-definitions": "^0.29.0",
     "@fluidframework/runtime-utils": "^0.29.0",
+    "@fluidframework/synthesize": "^0.29.0",
```

Comment:
nit: looks like this was part of a different change from the draft pr but not a big deal

Reply:
Thanks, easy to fix!